src/qibojit/backends/__init__.py

```diff
@@ -1,2 +1,44 @@
+from qibo.config import raise_error
+
 from qibojit.backends.cpu import NumbaBackend
 from qibojit.backends.gpu import CupyBackend, CuQuantumBackend
+
+
+class MetaBackend:
+    """Meta-backend class which takes care of loading the qibojit backends."""
+
+    def __init__(self):
+        self.platforms = ("numba", "cupy", "cuquantum")
```

Comment:
It is just a constant.
```suggestion
    PLATFORMS = ("numba", "cupy", "cuquantum")
```